vip_hci/var/__init__.py

```diff
@@ -9,5 +9,5 @@
 
 from .filters import *
 from .fit_2d import *
-from .shapes import *
 from .plotting import *
+from .shapes import *
```

Comment:
what is the purpose of this change in ordering?

Reply:
That is to order the imports alphabetically, like in the other submodules.